cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -139,32 +139,40 @@ parameter_types! {
 		deneb: Fork {
 			version: [4, 0, 0, 0], // 0x04000000
 			epoch: 0,
-		}
+		},
+		electra: Fork {
+			version: [5, 0, 0, 0], // 0x05000000
+			epoch: 2000000,
```

Comment:
Seems the Sepolia upgrade scheduler is not finalized, so this is just a placeholder in advance?

Reply:
Exactly, added a comment in https://github.com/paritytech/polkadot-sdk/pull/7075/commits/0b691062ff4307ed67221a26b117324c3bdbda9d.